common/djangoapps/auth_exchange/views.py

```diff
@@ -44,14 +55,72 @@ def post(self, request, _backend):  # pylint: disable=arguments-differ
         scope = form.cleaned_data["scope"]
         client = form.cleaned_data["client"]
 
+        return self.exchange_access_token(request, user, scope, client)
+
+    def exchange_access_token(self, request, user, scope, client):
+        raise NotImplementedError()
+
+
+class DOPAccessTokenExchangeView(AccessTokenExchangeBase, AccessTokenView):
```

Comment:
Suggestion: import `AccessTokenView` as `dop_TokenView` so it's not as subtle.
And similarly import `TokenView` as `dot_TokenView`.